core/summary_api.php

```diff
@@ -91,6 +95,85 @@ function summary_helper_get_developer_label ( $p_user_id ) {
 
 }
 
+/**
+ * Calculate bug status count according to  'open', 'resolved' and 'closed', 
+ * then put the numbers into $p_cache array
+ *
+ * @param array &$p_cache    The cache array.
+ * @param string $p_key      The key of the array.
+ * @param string $p_status   The status of issues.
+ * @param string $p_bugcount The bug count of $p_status issues.
+ * @return void
+ */
+function summary_helper_build_bugcount( &$p_cache, $p_key, $p_status, $p_bugcount ) {
+	$t_resolved_val = config_get( 'bug_resolved_status_threshold' );
+	$t_closed_val = config_get( 'bug_closed_status_threshold' );
+
+	if( $t_closed_val <= $p_status ) {
+		if( isset( $p_cache[$p_key]['closed'] ) ) {
+			$p_cache[$p_key]['closed'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['closed'] = $p_bugcount;
+		}
+	} else if( $t_resolved_val <= $p_status ) {
+		if( isset( $p_cache[$p_key]['resolved'] ) ) {
+			$p_cache[$p_key]['resolved'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['resolved'] = $p_bugcount;
+		}
+	} else {
+		if( isset( $p_cache[$p_key]['open'] ) ) {
+			$p_cache[$p_key]['open'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['open'] = $p_bugcount;
+		}
+	}
+}
+/** 
+ * Build bug links for 'open', 'resolved' and 'closed' issue counts
+ * 
+ * @param string $p_bug_link            The base bug link.
+ * @param string &$p_bugs_open          The open bugs count, return open bugs link.
+ * @param string &$p_bugs_resolved      The resovled bugs count, return resolved bugs link.
+ * @param string &$p_bugs_closed        The closed bugs count, return closed bugs link.
+ * @param string &$p_bugs_total         The total bugs count, return total bugs link.
```

Comment:
I think those 4 params are integers, not strings

Reply:
Nevermind my comment - it's _integer_ input and _string_ output, so might as well leave the PHPDoc as-is.

Reply:
dregad, sorry for respond late. After the crazy busy weekdays, I saw you have fix the requested, are there anymore need me to commit?